### PR TITLE
426 csv names

### LINKED ISSUE #426: Admin: CSV Download Names
### Description:

~Prevent admin users from appearing on downloaded CSVs.~
Give better names to downloaded CSVs.

### Acceptance Criteria:
- [ ] ~Admin users/userquizs don't appear on downloaded CSVs.~
- [ ] ~Admin userquizs are deleted upon being parsed in this way.~
- [ ] When an admin downloads the CSV from ____, name it as follows:*
  - [ ] User Management: `all-students_yyyy-mm-dd-hh-mm-ss`
  - [ ] Exam Management: `<exam-name>_yyyy-mm-dd-hh-mm-ss`

### Definition of Done:
- [x] AC met
- [x] PR linked
- [ ] Branch merged

### Notes
\* Or something better you think of - just make the name useful :)

### PR DESCRIPTION
**Pull request by alexmingikim**
**Issue**

The names of the downloaded csv files were not meaningful. 

Closes #426

**Solution**

Renamed csv files as described on project board. 

**Risks**

None. 

**Reviewers**

@Oculux314 
@nzkevc

### CODE REVIEW

File: client/src/utils/quizManagement.ts [hunk 1]
```diff
@@ -226,7 +225,31 @@ export const downloadUserQuizzesCsvQuery = async (
...
+      month +
+      '-' +
+      day +
+      '-' +
+      hour +
+      '-' +
+      minute +
+      '-' +
+      second +
+      '.csv'
```
Comment (johnchen383) [inline client/src/utils/quizManagement.ts:252]: Could be extracted into a util since its shared. No biggie though.

Comment (alexmingikim) [reply client/src/utils/quizManagement.ts:252]: I'll look in to it

Comment (alexmingikim) [reply client/src/utils/quizManagement.ts]: Done!

**Review by Oculux314 — APPROVED** [review]
I like this! Clean code too 👍

**Review by bcho892 — APPROVED** [review]

**Pull request merged**